server/channels/store/sqlstore/context.go

```diff
@@ -23,6 +25,12 @@ func WithMaster(ctx context.Context) context.Context {
 	return context.WithValue(ctx, storeContextKey(useMaster), true)
```

Comment:
We should add a comment that this function is now deprecated and ask users to use the new one. And when all instances of `WithMaster` have been replaced, we should make it a private function.